gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -420,5 +420,188 @@ public static ChangeRequestListOption sortOrder(SortingOrder order) {
     }
   }
 
-  // TODO(mderka) Add methods. Created issue #596.
+  /**
+   * Creates a new zone.
+   *
+   * <p>Returns {@link ZoneInfo} object representing the new zone's information. In addition to the
+   * name, dns name and description (supplied by the user within the {@code zoneInfo} parameter),
+   * the returned object will include the following read-only fields supplied by the server:
+   * creation time, id, and list of name servers.
+   *
+   * @throws DnsException upon failure
+   * @see <a href="https://cloud.google.com/dns/api/v1/managedZones/create">Cloud DNS Managed Zones:
+   * create</a>
+   */
+  ZoneInfo create(ZoneInfo zoneInfo);
```

Comment:
The second two methods are dropped. The first two cannot have a test---this is just a Dns interface without an implementation. All tested methods are part of Zone, not of Dns. These two methods can be called on the service only and they cannot be called on a Zone object. They will be tested within in DnsImpl.